west.yml

```diff
@@ -43,8 +43,9 @@ manifest:
     - name: civetweb
       revision: e6903b80c09d17cd1a8bb32e40080005558aad29
       path: modules/lib/civetweb
-    - name: esp-idf
-      revision: 6835bfc741bf15e98fb7971293913f770df6081f
+    - name: hal_espressif
+      west-commands: west/west-commands.yml
+      revision: c836ae28675a7ce8b4bdb75b7379fdccfe2731fb
       path: modules/hal/esp-idf
```

Comment:
should point to a new directory, not esp-idf, maybe just espressif like other HALs

Reply:
yes, thanks.